src/FlaUILibrary/keywords/property.py

```diff
@@ -64,24 +68,45 @@ def get_property_from_element(self, identifier, action, msg=None):
 
         """
         # pylint: enable=line-too-long
-
-        module = self._container.get_module()
-        element = module.get_element(identifier, msg=msg)
+        
         action_value = ""
-
         try:
             action_value = Property.Action[action.upper()]
-        except KeyError:
-            FlaUiError.raise_fla_ui_error(FlaUiError.InvalidPropertyArgument)
-
-        if action_value is (Property.Action.MAXIMIZE_WINDOW
-                            or Property.Action.MINIMIZE_WINDOW
-                            or action_value == Property.Action.NORMALIZE_WINDOW):
+        except KeyError or action_value in [Property.Action.MAXIMIZE_WINDOW, Property.Action.MINIMIZE_WINDOW, Property.Action.NORMALIZE_WINDOW]:
             FlaUiError.raise_fla_ui_error(FlaUiError.InvalidPropertyArgument)
-
-        return module.action(action_value,
-                             Property.create_value_container(element=element, uia=module.identifier()),
-                             msg)
+        
+        combobox_item_flag = self.__is_combobox_list_item(identifier)
+        if combobox_item_flag:
+            self.__prepare_combobox_element_for_selection_item_pattern(identifier)
+        
+        module = self._container.get_module()
+        element = module.get_element(identifier, msg=msg)
+        property_value = module.action(action_value,
+                            Property.create_value_container(element=element, uia=module.identifier()), 
+                            msg)
+        
+        if combobox_item_flag:
+            self.__prepare_combobox_element_for_selection_item_pattern(identifier)
+        
+        return property_value
+    
+    def __is_combobox_list_item(self, identifier):
+        patterns = identifier.split("/")
+        if "ComboBox" in identifier and "ComboBox" not in patterns[-1]:
+            return True
+        return False
+    
+    def __prepare_combobox_element_for_selection_item_pattern(self, identifier):
```

Comment:
Why is this method required?

Extend and Collapse is already implemented in combox.py

https://github.com/GDATASoftwareAG/robotframework-flaui/blob/main/src/FlaUILibrary/flaui/module/combobox.py

If you want to use them you have to call this Action to get an guarantee error handling. All keywords should not call any C# code here because this is why it's abstracted in a module action interface.